inventory/sample.casl.rht-labs.com.d/inventory/hosts

```diff
@@ -0,0 +1,71 @@
+[all:vars]
+ansible_user=openshift
+ansible_become=True
+
+# OpenStack Provisioning Variables
+openstack_ssh_public_key="ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC+A26nd6teDDU6WjR+PlMOK3jbi0kBg2wXh+g5wJ2TDxKIvFiL9vgejbsFmeCYPnkuu3Ib6BGB/1ctLycIHMPRby2vWd43MKDrg0dlUf8e0slb8u0mLUQ4iSpX9JGmGIrJYpgCeroeteeks0SFtmug3UbCITX88Q4OcpZ1KHN5P0Lmn/CNzopHgWue/ijOYvRfapAg7voiOmlGqY50smWX/KDcnwkKULkVZwNnouaq4iKOtJL/MFTUJaiPiFk4IcvZo1aAcdDEhjY1exVT5YowDFN84a+RLee4x9sR1dRV5iYO3Ta+gScds78rsHS8NXrZlkU/tYYgAs7nJABVoKIB esauer@etl-util3.etl.lab.eng.rdu2.redhat.com"
+openstack_default_image_name="rhel-guest-image-7.2"
+openstack_default_flavor="m1.large"
+openstack_external_network_name="external"
+openstack_dns_domain="casl.rht-labs.com"
```

Comment:
same as above - please change 'rht-labs.com' to 'example.com' (global search/replace)